sdk/formrecognizer/azure-ai-formrecognizer/tests/test_receipt_from_url_async.py

```diff
@@ -387,3 +387,30 @@ async def test_receipt_continuation_token(self, client):
             result = await poller.result()
             self.assertIsNotNone(result)
             await initial_poller.wait()  # necessary so azure-devtools doesn't throw assertion error
+
+    @GlobalFormRecognizerAccountPreparer()
+    @GlobalClientPreparer()
+    async def test_receipt_locale_default(self, client):
+        def _get_locale_in_call(pipeline_response, _, headers):
+            assert 'en-US' == pipeline_response.http_response.request.query['locale']
+        async with client:
+            poller = await client.begin_recognize_receipts_from_url(self.receipt_url_jpg, cls=_get_locale_in_call)
+            await poller.wait()
+
+    @GlobalFormRecognizerAccountPreparer()
+    @GlobalClientPreparer()
+    async def test_receipt_locale_specified(self, client):
+        def _get_locale_in_call(pipeline_response, _, headers):
+            assert 'en-IN' == pipeline_response.http_response.request.query['locale']
+        async with client:
+            poller = await client.begin_recognize_receipts_from_url(self.receipt_url_jpg, locale="en-IN", cls=_get_locale_in_call)
+            await poller.wait()
+
+    @GlobalFormRecognizerAccountPreparer()
+    @GlobalClientPreparer()
+    async def test_receipt_locale_error(self, client):
+        with pytest.raises(HttpResponseError) as e:
+            async with client:
+                await client.begin_recognize_receipts_from_url(self.receipt_url_jpg, locale="not a locale")
+        assert "locale" in e.value.error.message
```

Comment:
sorry last minute, but should we add a test here that test locale with api version 2.0?

Reply:
oof good point. i copied these over from business cards, where it was not necessary